pom.xml

```diff
@@ -893,8 +900,39 @@
             <groupId>io.netty</groupId>
             <artifactId>netty-transport-udt</artifactId>
           </exclusion>
+          <exclusion>
```

Comment:
Has been shaded to netty-common

https://github.com/netty/netty/blob/bed705183f555eca1113f1359800abf0da3d067b/common/pom.xml#L104-L132